src/api/hopperApi.ts

```diff
@@ -34,8 +35,9 @@ export class HopperApi extends ApiBase implements IHopperApi {
             xhr.onload = () => {
                 try {
                     let info = JSON.parse(xhr.responseText);
-                    this.instanceLoginUrl = info.loginUrl
-                    this.instanceApiRot = info.apiRoot
+                    this.instanceLoginUrl = info.loginUrl;
+                    this.instanceLoginUrl = info.logoutUrl;
+                    this.instanceApiRot = info.apiRoot;
```

Comment:
you are assigning the logout url to the instance's login url. i'm pretty sure this is not intended.